CONTRIBUTING.md

```diff
@@ -97,16 +97,26 @@ In addition to testing the Ruby parts out, you can also test the node package pa
 ```sh
 cd react_on_rails/
 yarn
-yarn run build
-yarn install-react-on-rails
+yarn run build:watch
+yalc pubish react-on-rails
```

Comment:
`publish`

Reply:
Thank you @tahsin352 for your review!!!!

@Judahmeek did we get all of these?